sycl/doc/design/spirv-extensions/SPV_INTEL_global_variable_decorations.asciidoc

```diff
@@ -207,7 +207,8 @@ Modify Section 3.31, Capability, adding a row to the Capability table:
 
 === Validation Rules
 
-None.
+* No two *HostAccessINTEL* decorations in the same module may have the same
+  _Name_ operand.
```

Comment:
This addition is good in principle, but I would consider rephrasing it using terms consistent with [Validity and Defined Behavior](https://www.khronos.org/registry/SPIR-V/specs/unified1/SPIRV.html#_validity_and_defined_behavior).  Maybe something like:

> * It is invalid for two **HostAccessINTEL** decorations in the same module to have the same _Name_ operand.

Is this a check that we will be adding to the SPIR-V validator?

Reply:
Thanks, that is better.  Done  in 19e5a2520f57d1e97761ec3238410cb1450237de.

> Is this a check that we will be adding to the SPIR-V validator?

Yes, I would presume so, but I'm not sure of the timing.  Do we have a fork of the SPIRV-Tools repo where we make vendor-specific additions like this?  If not, I  presume we will need to wait until the "SPV_INTEL_global_variable_decorations" extension document is promoted to the Khronos repo before submitting a PR to the Khronos "SPIRV-Tools".